README.md

```diff
@@ -10,6 +10,8 @@ transaction over a TCPSocket.
 To use the example with a different interface, you will need to modify main.cpp and
 replace the EthernetInterface class with the appropriate interface.
 
+**Note:** The default serial port baudrate is 9600 bauds.
```

Comment:
Query: Could we change "9600 bauds" to "9600" to get rid of the redundancy because we already said "baud rate"?

Reply:
I'd suggest we change it to `9600 bit/s`, as 9600 doesn't look good.